samples/oci-apigw-idcs-auth-basic/README.md

```diff
@@ -1,181 +1,204 @@
 
 Enter a name for your IDCS Application, for example "myAPI".
 
 ![IDCS-appcreate1](./images/IDCS-appcreate1.png)
 
-For "Allowed Grant Types", select "Resource Owner". Click *Next*.
+For "Allowed Grant Types", select "Resource Owner". Click _Next_.
 
 ![IDCS-appcreate2](./images/IDCS-appcreate2.png)
 
 For Primary Audience, enter anything "display-httprequest-info" for example.
-For Scopes, click *Add*. In the dialog box, for field "Scope", enter anything "display-httprequest-info" for example, click *Add*.
+For Scopes, click _Add_. In the dialog box, for field "Scope", enter anything "display-httprequest-info" for example, click _Add_.
 
 ![IDCS-appcreate3](./images/IDCS-appcreate3.png)
 
-Click *Next*.
+Click _Next_.
 
 ![IDCS-appcreate4](./images/IDCS-appcreate4.png)
 
-Click *Finish*.
+Click _Finish_.
 
 ![IDCS-appcreate5](./images/IDCS-appcreate5.png)
 
-Now that the application is added, note the *Client ID* and *Client Secret*.
+Now that the application is added, note the _Client ID_ and _Client Secret_.
 
 ![IDCS-appcreate6](./images/IDCS-appcreate6.png)
 
-Click *Close*.
+Click _Close_.
 
-Click on Configurations tab  under Client Information section click on add scope and select the *application name* from the dropdown. Note the scope value.
+Click on Configurations tab under Client Information section click on add scope and select the _application name_ from the dropdown. Note the scope value.
 
 ![IDCS-appcreate7](./images/IDCS-appcreate7.png)
 ![IDCS-appcreate8](./images/IDCS-appcreate8.png)
 
-Click *Activate* and click *Ok* in the dialog.
-
-Note the *IDCS URL*, this is the URL you see in your browser URL bar, copy the IDCS url ( For example: https://idcs-xxxxxxxxxxx.identity.oraclecloud.com/ ), client-id, client-secret and scope these values are provided to the Basicauth function.
-
+Click _Activate_ and click _Ok_ in the dialog.
 
+Note the _IDCS URL_, this is the URL you see in your browser URL bar, copy the IDCS url ( For example: https://idcs-xxxxxxxxxxx.identity.oraclecloud.com/ ), client-id, client-secret and scope these values are provided to the Basicauth function.
 
 ## Review and customize the function
+
 Review the following files in the current folder:
+
 - [pom.xml](./pom.xml) specifies all the dependencies for your function
 - [func.yaml](./func.yaml) that contains metadata about your function and declares properties
 - [src/main/java/com/example/fn/BasicAuth.java](./src/main/java/com/example/fn/BasicAuth.java) which contains the Java code
 
-The name of your function *basicauth* is specified in [func.yaml](./func.yaml).
+The name of your function _basicauth_ is specified in [func.yaml](./func.yaml).
+
+set the following config variables to the values noted while configuring IDCS. The IDCS URL is the token endpoint that returns the access token after validating credentials
 
-set the following variable in "src/main/java/com/example/utils/ResourceServerConfig.java" to the values  noted while configuring IDCS.
 ```
```

Comment:
```suggestion
```java
```

Reply:
This is not java, but the configuration variables in the function.

Reply:
Sure, but it probably could be formatting used Java syntax highlighting. This is a formatting improvement, so the language specified does not need to be accurate.